automated-comments/csharp/two-fer/use_single_formatted_string_not_multiple.md

```diff
@@ -0,0 +1 @@
+Currently, there is a bit of duplication in the two strings that are returned. Could you try to structure your code such that only one `string` is returned?
```

Comment:
@mikedamay said:

"I found “Currently, there is a bit of duplication in the two strings that are returned.” a little confusing. Maybe “Currently, there is a bit of duplication in the two strings in the return statment.”